lib/remap-functions/src/parse_url.rs

```diff
@@ -73,31 +72,38 @@ fn inner_type_def() -> Option<InnerTypeDef> {
 }
 
 fn url_to_value(url: Url) -> Value {
```

Comment:
I made these changes in response to https://github.com/timberio/vector/issues/5573#issuecomment-747309478